pkg/manifests/manifests_test.go

```diff
@@ -56,6 +63,30 @@ func TestManifests(t *testing.T) {
 		t.Errorf("invalid RouterClusterRoleBinding: %v", err)
 	}
 
+	if _, err := f.MetricsClusterRole(); err != nil {
+		t.Errorf("invalid MetricsRole: %v", err)
```

Comment:
Copy-and-paste error: The method is named `MetricsClusterRole`, and the error message says "MetricsRole".